src/richie/apps/courses/templates/courses/cms/organization_detail.html

```diff
@@ -26,7 +26,7 @@ <h1 class="organization-detail__title">{{ current_page.get_title }}</h1>
         </div>
 
         <ul class="organization-detail__content__courses">
-          {% for course in organization.courses.drafts %}
+          {% for course in organization.courses.all %}
```

Comment:
Substituting "all" for "drafts" in the templates, list all courses including the one still in draft and the unpublished as well. The title won't appear because we aren't referencing the right "get.title" but they are in the list nonetheless.